python/ml_croissant/scripts/serialize_from_json.py

```diff
@@ -0,0 +1,44 @@
+r"""Example script to serialize a JSON to valid Croissant JSON-LD.
+
+# noqa: E501
+# pylint: disable=line-too-long
+Usage:
+python scripts/serialize_from_json.py \
+    --json "{\"@type\": \"sc:Dataset\", \"name\": \"PASS\", \"description\": \"PASS is a large-scale image dataset that does not include any humans and which can be used for high-quality pretraining while significantly reducing privacy concerns.\", \"@language\": \"en\", \"license\": \"https://creativecommons.org/licenses/by/4.0/\", \"url\": \"https://www.robots.ox.ac.uk/~vgg/data/pass/\"}"
+# pylint: enable=line-too-long
+# qa: E501
+"""
+
+import json
+
+from absl import app
+from absl import flags
+from absl import logging
+
+import ml_croissant as mlc
+
+flags.DEFINE_string(
+    "json",
+    None,
+    "JSON representation to convert to valid Croissant JSON-LD.",
+)
```

Comment:
A possible future improvement might be to provide a file to load (instead of json.loads a json)? Might be easier for the users than typing a long json?

Reply:
Done.

You're very right about this... It cannot be used as is. I will simply delete this whole script for now, as I haven't thought it through yet.